README.md

```diff
@@ -264,6 +264,10 @@ When developing, you can use the `Makefile` for doing the following operations:
 If you are working in a large organization you should learn more about [Kong
 Enterprise](https://konghq.com/kong-enterprise-edition/).
 
+## Contributions
```

Comment:
this should not be its own section, but should go at the top of the "Development" section